src/ComparisonContainer/ComparisonContainer.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import ComparisonDetails from '../ComparisonDetails/ComparisonDetails';
+import './ComparisonContainer.scss';
+
+const ComparisonContainer = () => {
+  return(
+    <section>
+      <h3>How Does It Compare?</h3>
+      <section>
+        <section>
+          <p>Write up text explaining app</p>
+        </section>
+        <select>
```

Comment:
We will need to add an `onClick` and `onChange` function to the select drop down to give it functionality.

Reply:
I like how responsive this list will be across devices.